packages/fetch-cached-dns/package.json

```diff
@@ -3,32 +3,43 @@
   "version": "2.1.0",
   "description": "A decorator on top of `fetch` that caches the DNS query of the `hostname` of the passed URL",
   "license": "MIT",
+  "main": "index.js",
+  "types": "index.d.ts",
   "files": [
     "index.js",
     "index.d.ts",
     "util.js"
   ],
   "scripts": {
-    "test": "jest test"
+    "test": "jest test",
+    "lint": "cross-env TIMING=1 eslint *.js",
```

Comment:
What is TIMING?

Reply:
https://turborepo.org/docs/features/caching#configuring-cache-outputs

Reply:
Why do you need `cross-env` here?

Reply:
Windows CI environment 😢